### DISCUSSION TITLE
Preprocess audio with MDX23 Kim_vocal_v2 vocal extraction model.

### DISCUSSION THREAD

**Discussion by dandanarsk** [McCloudS/subgen#90] [Ideas]
Can the MDX23 Kim_vocal_v2 vocal extraction model be integrated on this, it looks more performant on my Windows standalone installation of @Purfview [https://github.com/Purfview/whisper-standalone-win?tab=readme-ov-file](url)?

**Comment by McCloudS** [comment]
Probably not, unless stable-ts supports it.  Other issue is whisper standalone has no source available and searching “Kim_vocal_v2” returns nothing except its own result.
